### PR TITLE
Add documentation on changing rule severity

### PR DESCRIPTION
**Pull request by CptWesley**
Related to #86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,26 @@ To add a props file:
 </Project>
 ```
 
+## Adjusting severity
+In order to adjust the severity of certain rules it is possible to use a `.globalconfig` file.
```
Comment (Corniel) [inline README.md:67]: Please mention where to put this file. Mention that .editorconfig will not work (with a link to the reason why).

**Review by Corniel — CHANGES_REQUESTED** [review]
As mentioned, please extend the documentation.

File: README.md (hunk 1 above)
Comment (CptWesley) [reply README.md]: Updated

**Review by Corniel — APPROVED** [review]
LGTM!

**Pull request merged**